src/components/Header/Header.spec.js

```diff
@@ -0,0 +1,10 @@
+import {render, screen} from '@testing-library/react';
+
+import Header from './Header';
+
+describe('Header', () => {
+  it('Shows that the header component is rendered', () => {
+    render(<Header />);
+    screen.getByAltText('mov.me logo');
+  });
```

Comment:
You are missing something here to complete your test:

```suggestion
  it('renders header with app logo', () => {
    render(<Header />);
    const logo = screen.getByAltText('mov.me logo');
    
    expect(logo).toBeInTheDocument()
  });
```

Reply:
This was pretty hidden in the documentation... :)
https://testing-library.com/docs/guide-disappearance/#nottobeinthedocument'